tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -325,11 +327,17 @@ def test_dhcp_relay_default(ptfhost, dut_dhcp_relay_data, validate_dut_routes_ex
                 if testing_mode == DUAL_TOR_MODE:
                     standby_duthost = rand_unselected_dut
                     start_dhcp_monitor_debug_counter(standby_duthost)
+                    client_iface = str(dhcp_relay['client_iface']['name'])
+                    # In case of active-standby ports northbound  trafic is duplicated
+                    # to both ToRs by the y-cable whereas is case of active-active
+                    # ports the traffic is ECMPd, therefore only one ToR receives
+                    # northbound packets
+                    discReqCount = 0 if client_iface in active_active_ports else 1
```

Comment:
The traffic is always forwarded to the active side, right? With PR https://github.com/sonic-net/sonic-mgmt/pull/11921, there is no counter increase on the standby side.

Reply:
Yes that is correct but #11921 was for active-active dualtor, and this PR doesn't fix the test for active_active ports that's why the expected count ( `discReqCount` ) is 0 in case of active_active ports.

In active-standby dualtor northbound the traffic is still duplicated to both ToRs.

Reply:
Yes, that makes sense.